src/components/four-column-block.js

```diff
@@ -50,7 +51,9 @@ const FourColumnBlock = () => {
       }
     }
   `)
+  const intl = useIntl()
   const blockData = data.contentJson.four_column_block
```

Comment:
Every instance where blockData is being called below needs the intl magic. 

four-column-block is the object in JSON (look for '"four-column-block" in en.json, es.json )

For example, line 64:

{blockData.description1} needs to turn into            {intl.formatMessage({id: `four-column-block.description`,})}

and so on for the rest of the content in this block

Reply:
I'm going to need a hand here. `{intl.formatMessage({id: four-column-block.description,})}` is not building. On save it creates spaces between four, column, block and its dashes. The Error it gaves says that four, column, and block aren't defined.

Reply:
I got this to build, but I'm not sure what the result is supposed to be.